docs/01-overview/02-main-areas/service-management/smgt-13-hb-test-addons.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Test addons
```

Comment:
same here - is this instructional? Are there specific steps that need to be followed?

Reply:
Changed to tutorial with the same title